webpack.config.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
       'bundle-[name].js' :
       'bundle-[name]-[chunkhash].js',
     path: path.join(__dirname, '/public/js'),
-    publicPath: __DEV__ ? 'http://localhost:2999/js' : '/js'
+    publicPath: __DEV__ ? `http://localhost:${process.env.PORT || 3000}/js` : '/js'
```

Comment:
Because it is served by the webpack dev server not the main server. That is why it is on a different port.